torch/_inductor/dependencies.py

```diff
@@ -637,12 +637,15 @@ def extract_input_node_reduction_ranges(
     Otherwise returns (None, None).
     """
 
-    from .ir import ComputedBuffer, Loops
+    from .ir import ComputedBuffer, ExternKernel, Loops
+
+    size: Optional[List[sympy.Expr]]
+    reduction_size: Optional[List[sympy.Expr]]
 
     if isinstance(input_node.data, ComputedBuffer):
         # Input node has already been realized. Return its size and reduction_size.
-        size = input_node.get_size()
-        reduction_size = input_node.get_reduction_size()
+        size = [*input_node.get_size()]
+        reduction_size = [*input_node.get_reduction_size()]
```

Comment:
Curious, is this equivalent to `list(...)`?

Reply:
Yeah curious what the motivation here is for this. It should actually be marginally less efficient than the ctor list method, shouldn't it?

Reply:
It is faster (for small lists) and looks cleaner IMO:
```
>>> timeit.timeit("list(range(10))")
0.12816928839311004
>>> timeit.timeit("[*range(10)]")
0.11478627705946565
>>> 
```

`list` gets mapped to a `LOAD_GLOBAL` bytecode (since you could do `list = tuple` in global scope), while `[*x]` gets mapped to list-specific bytecodes without touching global scope.